blog/app/Http/Requests/TagRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class TagRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'tag_name' => 'required',
+        ];
+    }
+
+    public function messages()
+    {
+        return [
+            'tag_name.required' => 'The tag name is required',  
+        ];
+    }
+
+    public function attributes()
+    {
+        return [
+            'tag_name' => 'Tag name',
+        ];
+    }
```

Comment:
https://laravel.com/docs/7.x/validation#customizing-the-validation-attributes
A khai báo thế này nhưng nó có được dùng ko a. Đọc kỹ hướng dẫn sử dụng trước khi dùng. :D